turbinia/workers/gitlab.py

```diff
@@ -0,0 +1,163 @@
+
+    Returns:
+        TurbiniaTaskResult object.
+    """
+    # Where to store the resulting output file.
+    output_file_name = 'gitlab_analysis.txt'
+    output_file_path = os.path.join(self.output_dir, output_file_name)
+    # Set the output file as the data source for the output evidence.
+    output_evidence = ReportText(source_path=output_file_path)
+
+    reports = []
+    summaries = []
+
+    # Grep for exif in workhorse logs
+    (r, priority, s) = self._is_exif_in_logs(
+        result, evidence.local_path,
+        os.path.join('var', 'log', 'gitlab', 'workhorse.log'))
+    if r != '':
+      reports.append(r)
+    if s != '':
+      summaries.append(s)
+
+    (r, p2, s) = self._is_exif_in_logs(
+        result, evidence.local_path,
+        os.path.join('var', 'log', 'gitlab', 'gitlab-workhorse', '@*'))
+    if r != '':
+      reports.append(r)
+    if s != '':
+      summaries.append(s)
+    if p2 < priority:
+      priority = p2
+
+    # TODO: Check for Metasploit Module
+    # 'https://packetstormsecurity.com/files/160441/GitLab-File-Read-Remote-Code-Execution.html'
+
+    (r, p3, s) = self._is_traversal_in_logs(
+        result, evidence.local_path,
+        os.path.join('var', 'log', 'gitlab', 'nginx', '*access*'))
+    if r != '':
+      reports.append(r)
+    if s != '':
+      summaries.append(s)
+    if p3 < priority:
+      priority = p3
+
+    if priority == priority.LOW:
+      result.close(self, success=True, status='No Gitlab exploitation found')
+      return result
+
+    report = "\n".join(reports)
+    summary = "\n".join(summaries)
+
+    output_evidence.text_data = report
+    result.report_priority = priority
+    result.report_data = report
+
+    # Write the report to the output file.
+    with open(output_file_path, 'wb') as fh:
+      fh.write(output_evidence.text_data.encode('utf-8'))
+
+    # Add the resulting evidence to the result object.
+    result.add_evidence(output_evidence, evidence.config)
+    result.close(self, success=True, status=summary)
+    return result
+
+  def _is_exif_in_logs(self, result, basedir, logfiles):
+    """Checks to see if there is evidence of the exiftool exploit in the logs.
+
+    Args:
+      result (TurbiniaTaskResult): The object to place task results into.
+      basedir (str): the root of the evidence.
+      logdir (str): The file(s) to check
+
+    Returns:
+      Tuple(
+        report_text(str): The report data
+        report_priority(int): The priority of the report (0 - 100)
+        summary(str): A summary of the report (used for task status)
+      )
+    """
+
+    check = " ".join(glob.glob(os.path.join(basedir, logfiles)))
+    if not len(check):
+      return ('', Priority.LOW, '')
+
+    cmd = ['zgrep', '"exiftool command failed"', check]
+    ret, result = self.execute(cmd, result, success_codes=[0, 1])
+    if ret == 0:
+      summary = 'exif exploit detected in {}'.format(logfiles)
```

Comment:
Minor nit, but for codebase consistency (I think it's in the style guide too) can update the format string to `{0:s}`?